lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -24,9 +24,9 @@
 use Doctrine\Common\Collections\Expr\Comparison;
 use Doctrine\Common\Util\ClassUtils;
 use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Driver\ResultStatement as DriverStatement;
 use Doctrine\DBAL\LockMode;
 use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Statement;
```

Comment:
Unsure if we should wait for https://github.com/doctrine/dbal/pull/4596 and revert this change in use statements. I think that change might be good because more forward compatible.

Reply:
I'll leave you decide. But to me, such change seems a bit unrelated to this PR